packages/upload/src/upload.vue

```diff
@@ -92,6 +92,10 @@ export default {
       if (before && before.then) {
         before.then(processedFile => {
           const fileType = Object.prototype.toString.call(processedFile);
+
+          processedFile.name = rawFile.name;
```

Comment:
Is this line necessary? Because here `processedFile` already has a `name` prop.

Reply:
A [blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) object only has two properties: `size` and `type`. In the given demo, the `name` property is assigned by the image compressor.

Reply:
OK

Reply:
This change will cause an error if here the processedFile is still a File object.